### PR TITLE
Make SwipeRefreshLayout look more like other bindings

### PR DESCRIPTION
**Pull request by Cheesebaron**

### CODE REVIEW

**Comment by martijn00** [comment]
Looks fine to me!

File: Cirrious.MvvmCross.Droid.Support.V4/MvxSwipeRefreshLayout.cs [hunk 1]
```diff
@@ -1,46 +1,56 @@
...
+	[Register("cirrious.mvvmcross.droid.support.v4.MvxSwipeRefreshLayout")]
+	public class MvxSwipeRefreshLayout : SwipeRefreshLayout 
+	{	 
+		protected MvxSwipeRefreshLayout(IntPtr javaReference, JniHandleOwnership transfer)
+            : base(javaReference, transfer) { }
+
+		public MvxSwipeRefreshLayout(Context p0)
+			: this (p0, null) { }
+
+		public MvxSwipeRefreshLayout(Context p0, IAttributeSet p1)
```
Comment (jamie94bc) [inline Cirrious.MvvmCross.Droid.Support.V4/MvxSwipeRefreshLayout.cs]: Can we have these parameters named too? :smile:

**Comment by Cheesebaron** [comment]
Better now @jamie94bc?

**Comment by jamie94bc** [comment]
Yeah! :smiley:

**Pull request merged**